<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_2</class>
 <widget class="QMainWindow" name="Game_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game_2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>9</y>
      <width>405</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_new_game">
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_answer">
       <property name="text">
        <string>Answer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_check">
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_restart">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_dimension">
       <item>
        <property name="text">
         <string>6x6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8x8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10x10</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lb_info">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
